### PR TITLE
chore(multiple-arbitrable-transaction): set RAB pragma comment

### PR DESCRIPTION
**Pull request by n1c01a5**
and remove the restriction to call the timeout (@mtsalenc proposal)

### CODE REVIEW

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol [hunk 1]
```diff
@@ -1,18 +1,18 @@
 /**
- * @authors: []
- * @reviewers: []
+ * @authors: [@clesaege, @n1c01a5, @epiqueras, @satello, @ferittuncer, @eburgos, @dannybabbev]
```
Comment (epiqueras) [inline contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: I didn't write this contract, pretty sure the only authors are "eburgos" and you, @n1c01a5.

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+ * @reviewers: [@clesaege, @n1c01a5, @epiqueras, @satello, @ferittuncer, @mtsalenc, @yushih]
```
Comment (epiqueras) [inline contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: You should not add another person's name to the reviewers list. This signifies they approve the contract as is and deem it vulnerability free to the best of their knowledge. They have to do it themselves.

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol (hunk 1 above)
Comment (n1c01a5) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: ok but could we have a contributors parameter?

Comment (epiqueras) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: `authors === contributors`

Actually, looks like more of us authored it.

See history: https://github.com/kleros/kleros-interaction/commits/66166fb63ebf71d5aa45412311b693d591921334/contracts/standard/arbitration/MultipleArbitrableTransaction.sol

**Comment by n1c01a5** [comment]
@epiqueras updated

File: contracts/standard/arbitration/MultipleArbitrableTransaction.sol (hunk 1 above)
Comment (n1c01a5) [reply contracts/standard/arbitration/MultipleArbitrableTransaction.sol]: yeah that's reason before I added some authors. Anyway they can do a PR

**Pull request merged**